### ISSUE TITLE
Build ANGLE binaries with /APPCONTAINER

### ISSUE THREAD

**Issue by jdm** [servo/servo#24892]
Right now we fail the AppContainerCheck test in the WACK security features tests.

**Comment by jdm** [comment]
The following ANGLE patch should address the problem (and will need to be upstreamed):
```
diff --git a/BUILD.gn b/BUILD.gn
index 036bf68b4..8305603a7 100644
--- a/BUILD.gn
+++ b/BUILD.gn
@@ -1085,7 +1085,11 @@ foreach(is_shared_library,
   }

   target(library_type, library_name) {
-    sources = util_sources
+    if (!angle_is_winuwp) {
+      sources = util_sources
+    } else {
+      sources = []
+    }
     deps = [
       ":angle_common",
       ":angle_util_loader_headers",
@@ -1152,8 +1156,12 @@ foreach(is_shared_library,
     if (is_shared_library) {
       defines = [ "LIBANGLE_UTIL_IMPLEMENTATION" ]

-      if (is_win && !angle_is_winuwp) {
-        sources += util_win_shared_sources
+      if (is_win) {
+        if (angle_is_winuwp) {
+          ldflags = [ "/APPCONTAINER", "/IGNORE:4010" ]
+        } else {
+          sources += util_win_shared_sources
+        }
       }

       public_deps += [ ":angle_util_loader" ]
```

**Comment by jdm** [comment]
Fixed by #25219.
